.github/workflows/operator-integration-test.yml

```
@@ -68,24 +68,43 @@ jobs:
             echo "Pod is created test case: PASS"
           fi
 
-      - name: Test for default instrumentation resources
+      - name: Test for default instrumentation resources for Java
         run: |
-          cd integration-tests/manifests
-          kubectl apply -f ./sample-deployment.yaml
+          kubectl apply -f integration-tests/java/sample-deployment-java.yaml
           sleep 60
```

```suggestion
          sleep 5
```
prob can be consistent with the 5s